### ISSUE TITLE
Autocomplete: Completion pops up twice on android+firefox

### ISSUE THREAD

**Issue by cerebralfornication** [jquery/jquery-ui#2120]
1. what I did: in firefox 105.1 on android 11, went to https://jqueryui.com/autocomplete/ , typed f in the text field, clicked "Fortran" in the completion list
2. what I expected: Fortran completed in the text field, autocompleter gone
3. what happened instead: Fortran in the text field but autocompler pops up again with "Fortran" as the only choice

works fine on desktop or on android+chrome.

**Comment by mgol** [comment]
Thanks for the report. Does the issue you describe exist when jQuery UI 1.12.1 is used or only with jQuery UI 1.13.0 or newer?

**Comment by cerebralfornication** [comment]
Yes, the problem also exists in 1.12.1, which is the version we use. The example url above uses 1.13.0 and shows it, too.

**Comment by mgol** [comment]
Thanks for the info. Since the issue is already in 1.12, given limited team resources it's not likely to be fixed by the UI team; see the project status at https://blog.jqueryui.com/2021/10/jquery-maintainers-update-and-transition-jquery-ui-as-part-of-overall-modernization-efforts/. PRs are welcome if they're not too complex.
